src/App.js

```diff
@@ -1,25 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Categories from './components/Categories';
+import Books from './components/Books';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <Header />
+    <Routes>
+      <Route path="/categories" element={<Categories />} />
+      <Route path="/books" element={<Books />} />
```

Comment:
- [ ] [OPTIONAL] Please make books your default Route, `<Route exact path="/books" element={<Books />} />` so as to avoid a blank page when we run the app